docs/refarch/boundedContexts/security/index.md

```diff
@@ -1,21 +1,71 @@
-# {name} BC
+# Security BC
+
+1. Overview
+2. User / Operator login (AuthN)
+3. Authorization Model (AuthZ)
+4. BC bootstrap 
+5. BC startup
+6. Role / Privilege association
+7. Example Query / call
 
-{overview}
 
 ## Terms
 
-Terms with specific and commonly accepted meaning within the Bounded Context in which they are used.
+Terms with specific and commonly accepted meaning within the Security Bounded Context.
 
-| Term | Description |
+| Crypto providers | Adapter that will provide cryptographic services and Key Management System |
 |---|---|
-| Term1 | Description1 |
+| IAM | Identity and Access Management. Adapter that will provide services for menus, profiles, roles and permissions.  |
+
+
+## Overview
+
+The protocol is request and response based, and the used transport protocol is secure Hypertext Transfer Protocol Secure (HTTPS). All services use the HTTP POST and GET methods. Both the request and response bodies are encoded in JSON-formatted text.
+
 
 ## Use Cases
 
-### Perform Transfer (universal mode)
+### User / Operator login (AuthN)
+
+The login function requires the user id and a secret key passed in the http body. The response contains a signed JWT token. The signature is generated by the Crypto subsystem.
+The login is performed by the Authorization services or IAM.
+
+![Use Case - Example REPLACE ME](./assets/securityBCv0.3.png)
+
+### Authorization Model (AuthZ)
+
+IAM will provide users / groups, roles and privileges associations. Each BC will also have a list of related roles. When a function or microservice
+is called, the role provided in the JWT will be compared with the roles associated with the BC, after signature verfication. 
+
+![Use Case - Example REPLACE ME](./assets/securityBCv0.8.png)
+
+### BC Bootstrap 
+
+At bootstrap, BC will send list of possible privileges. This is done once per deployement of a new version.
+
+![Use Case - Example REPLACE ME](./assets/securityBCv0.4.png)
+
+### BC Startup 
+
+At startup the BC will request authentication issuer public keys from Security BC Crypto / KMS subsystems and the list of roles / privileges Security BC IAM subsystem. A local crypto library signature verification function will verfiy the JWT signature and the roles in the JWT will be compared with the local list of roles obtained from the central authorization service. 
+
+
+![Use Case - Example REPLACE ME](./assets/bc_start_up.png)
+
+### Role / Privilege association 
+
+Roles are associated with a number of privileges.
+
+![Use Case - Example REPLACE ME](./assets/securityBCv0.6.png)
```

Comment:
Please change the info text for the image to something more useful then the default template `Use Case - Example REPLACE ME`.